packages/SwingSet/test/upgrade/vat-ulrik-2.js

```diff
@@ -46,6 +67,6 @@ export const buildRootObject = (_vatPowers, vatParameters, baggage) => {
     },
     getNewDurandal: () => newDur,
   });
-  // exercise async return
+  // exercise async return // XXX WHY???
```

Comment:
`buildRootObject()` is now allowed to return a Promise, as long as it fulfills promptly. We needed this to allow ZCF to do an `importBundle(contractBundle)` during upgrade, and `importBundle` returns a prompt-Promise. This line ensures that we can return such a Promise without errors: it was easier and faster than writing a whole new test for that one feature. But it sounds like I should add a comment to that effect here.

Could you replace my terse comment with the following?:
```
// buildRootObject() is allowed to return a Promise, as
// long as it fulfills promptly (we added this in #5246 to allow ZCF
// to use the async importBundle() during contract upgrade).
// We return a pre-fulfilled Promise here to exercise this ability.
```